Homework-№1-from-07.09.2022/task4.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#define systemOfCounting 10
+#define sizeOfArray (3 * systemOfCounting - 2)
+
+void task4(int *luckyTicketsCount)

```

Comment:
```suggestion
int amountOfLuckyTickets(int *luckyTicketsCount)
```
1) Более информативное название
2) А зачем так сложно результат возвращать?